kt-code-generators/src/test/kotlin/io/vrap/codegen/kt/languages/TestCodeGenerator.kt

```diff
@@ -22,7 +23,7 @@ class TestCodeGenerator {
 
     val generatorConfig = CodeGeneratorConfig(
             packagePrefix = "com.commercetools.importer",
-            ramlFileLocation = URI.createFileURI("../api-spec/api.raml")
+            ramlFileLocation = URI.createFileURI(Paths.get("").resolve("../api-spec/api.raml").toAbsolutePath().toString())
```

Comment:
Could it maybe better to use `System.getProperty("user.dir")` instead to get the current working dir?

Reply:
@jenschude This code didn't work on Marcels windows machine 😞  But I will give your suggestion a try on my computer at home.